styleguide/components.mdx

```diff
@@ -35,6 +35,52 @@ Decorator for external links
   deno.com
 </comp.ExternalLink>
 
+## Hero
+
+A hero section for the top of a page
+
+```tsx
+<comp.Hero  bgImage="deno-looking-up.svg">
```

Comment:
double space here, but also this image doesn't appear in the example. Does it need a fix for the path to the image?

```suggestion
<comp.Hero bgImage="deno-looking-up.svg">
```